CMakeLists.txt

```diff
@@ -654,6 +660,7 @@ add_custom_command (TARGET lz4 POST_BUILD
     ${CMAKE_BINARY_DIR}/lz4/include
   COMMAND ${CMAKE_COMMAND} -E copy
     $<TARGET_FILE:lz4> ${CMAKE_BINARY_DIR}/lz4/lib)
+exclude_if_included (lz4)
```

Comment:
We're excluding targets based on whether this is an in source build or an out of source build? That seems strange. I suspect I'm confused. Is that what this is doing or somthing else?

Reply:
see above. To be fair, there might still be installation bugs when we are vendor'd, but I believe this exclusion is correct.